odoo/addons/base/i18n/base.pot

```diff
@@ -16564,7 +16564,7 @@ msgstr ""
 
 #. module: base
 #: model:ir.module.module,shortdesc:base.module_l10n_din5008_industry_fsm
-msgid "DIN 5008 - Field Service"
+msgid "DIN 5008 Field Service Report Layout"
```

Comment:
no need to alter the .pot file in master. 🙂

Reply:
i have discussed with @ppr-odoo , 
@xavierbol as it is `module_l10n_din5008_industry_fsm` module .. no need to update the .pot file?
for `l10` modules we have to do the changes manually right?
am I missing anything?